crates/ruff_linter/src/rules/flake8_builtins/rules/stdlib_module_shadowing.rs

```diff
@@ -11,34 +11,35 @@ use crate::package::PackageRoot;
 use crate::settings::types::PythonVersion;
 
 /// ## What it does
-/// Checks for modules that use the same names as Python builtin modules.
+/// Checks for modules that use the same names as Python standard-library
+/// modules.
 ///
 /// ## Why is this bad?
-/// Reusing a builtin module name for the name of a module increases the
-/// difficulty of reading and maintaining the code, and can cause
-/// non-obvious errors, as readers may mistake the variable for the
-/// builtin and vice versa.
+/// Reusing a standard-library module name for the name of a module increases
+/// the difficulty of reading and maintaining the code, and can cause
+/// non-obvious errors. Readers may mistake the first-party module for the
+/// standard-library module and vice versa.
 ///
-/// Builtin modules can be marked as exceptions to this rule via the
+/// Standard-library modules can be marked as exceptions to this rule via the
 /// [`lint.flake8-builtins.builtins-allowed-modules`] configuration option.
```

Comment:
Do we need to update the settings name as well? (`lint.flake8-builtins.builtins-allowed-modules`) I think not but thought raise it regardless. Aside, the double "builtins" is a bit unfortunate.

Reply:
Hmm, good catch. I think we should

Reply:
I'd suggest to avoid the rename here and directly rename it to remove the "builtins" prefix from both options. I'll open a tracking issue but happy to hear any suggestions if you think otherwise.

Reply:
https://github.com/astral-sh/ruff/issues/15368